department-of-veterans-affairs/caseflow
# Return AMA appeals in queue case search

Connects #5766.

Return AMA appeals (if they exist) when searching for a Veteran's file in queue case search.

## Test plan
* Fire up the rails console `rails c` in the caseflow directory
* Create one or more AMA appeal records in the database using FactoryBot `FactoryBot.create(:appeal, veteran_file_number: file_num)` (You can use file numbers for cases that exist in our VACOLS test data set in order to see both types of appeal showing up in the same results list!)
* See the new records you created show up in the search results!
